sdk/cognitiveservices/ms-azure-cs-inkrecognizer/pom.xml

```diff
@@ -1,59 +1,54 @@
-<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
 <project xmlns="http://maven.apache.org/POM/4.0.0"
          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
+    <artifactId>azure-cognitiveservices-inkrecognizer</artifactId>
+    <groupId>com.microsoft.azure.cognitiveservices</groupId>
+    <version>1.1.0-preview.1</version>
```

Comment:
This is a new module. Why is the version `1.1.0` instead of `1.0.0`?

Reply:
Yes this should be `1.0.0-preview.1` if it is a new module.